ts/components/messages/EmptyListComponent.tsx

```diff
@@ -0,0 +1,40 @@
+import { Text, View } from "native-base";
+import * as React from "react";
+import { Image, ImageSourcePropType, StyleSheet } from "react-native";
+import customVariables from "../../theme/variables";
+
+const styles = StyleSheet.create({
+  view: {
+    padding: customVariables.contentPadding,
+    alignItems: "center"
+  },
+  title: {
+    paddingTop: customVariables.contentPadding
+  },
+  subtitle: {
+    textAlign: "center",
+    paddingTop: customVariables.contentPadding,
+    fontSize: customVariables.fontSizeSmall
+  }
+});
+
+type Props = Readonly<{
+  image: ImageSourcePropType;
+  title: string;
+  subtitle?: string;
+}>;
+
+export class EmptyListComponent extends React.Component<Props> {
```

Comment:
prefer functional component instead of class ones
(see [here](https://medium.com/@amarchitelli1993/react-functional-vs-class-components-ac6b0c9e38bb))

```typescript
export const EmptyListComponent = (props: Props) => {
  return (
    <View style={styles.view}>
      <View spacer={true} />
      <Image source={props.image} />
      <Text style={styles.title}>{props.title}</Text>
      {props.subtitle && <Text style={styles.subtitle}>{props.subtitle}</Text>}
    </View>
  );
};
```

Reply:
Updated within https://github.com/pagopa/io-app/pull/1513/commits/b3da2e84abc240b74b99cdca88857fdd69c97222.